src/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+set(Montage_SRC
+  itkParseTileConfiguration.cxx
+  )
```

Comment:
In the top level CMakeLists.txt file, we also need to add

```
set(Montage_LIBRARIES Montage
```

per https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/CMakeLists.txt#L4